drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c

```diff
@@ -170,6 +170,8 @@ enum wifi_nrf_status umac_cmd_init(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 
 	umac_cmd_data->op_band = op_band;
 
+	umac_cmd_data->mgmt_buff_offload = CONFIG_BSS_IDLE_MGMT_OFFLOAD;
```

Comment:
This is wrong, IIRC, `mgmt_buff_offload` was for whether UMAC needs RX buffers from host or not. This is for BSS IDLE management.

Reply:
True. I read this wrong. I will remove this for now since BSS_MAX_IDLE_TIME_OFFLOAD is yet to be supported in UMAC.